packages/builders/src/index.ts

```diff
@@ -1,6 +1,7 @@
 export * as EmbedAssertions from './messages/embed/Assertions.js';
 export * from './messages/embed/Embed.js';
-export * from './messages/formatters.js';
+// TODO: Consider removing this dep in the next major version
+export * from '@discordjs/formatters';
```

Comment:
We can make a follow-up PR once this one merges, to remove it, so we can include that change once we start preparing v15.